src/data/online.json

```diff
@@ -491,5 +491,21 @@
 		"author": "albinowax",
 		"notes": "Web application hacking game via missions, based on real vulnerabilities.",
 		"badge": null
-	}
+	},
+	{
+		"url": "https://cyberscavengerhunt.com",
+		"name": "Cyber Scavenger Hunt",
+		"technology": [
+			"Javacript",
+			"React"
+		],
+		"references": [
+			{
+				"name": "GitHub",

```

Comment:
Should be one of these enum values:
https://github.com/OWASP/OWASP-VWAD/blob/6628545a81071b32ce9da00d312298b4dc586c19/schema.json#L91-L102